### ISSUE TITLE
[NBug] startIndex cannot be larger than length of string.Paramet...

### ISSUE THREAD

**Issue by Broxzier** [gitextensions/gitextensions#7342]
## Current behaviour

Error occurs every time on rightclicking in the root folder of a cloned repository, then going to File History in the Git Extensions shell menu. Doing the same on a file or subfolder works as expected.

## Expected behaviour

The history window opens, showing the history of the entire repository (since we're in the root).

## Steps to reproduce

Go to the root folder of a cloned repository.
In the right-click shell menu, go to Git Extensions > File History.
Exception gets thrown.

## Error Details
```
System.ArgumentOutOfRangeException: startIndex cannot be larger than length of string.
Parameter name: startIndex
   at System.String.Substring(Int32 startIndex, Int32 length)
   at GitUI.GitUICommands.RunFileHistoryCommand(IReadOnlyList`1 args)
   at GitExtensions.Program.RunApplication()
   at GitExtensions.Program.Main()
```

## Additional information
Tested on two local repositories. One old and one freshly cloned.

## Environment
- Git Extensions 3.2.1.6628
- Build bc07663c3c893f1d896520e0bdcfbe35cfe4b729
- Git 2.22.0.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3468.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
This looks like may have been fixed in #7109
Please try the latest dev build and see whether the issue has been fixed.

**Comment by Broxzier** [comment]
Tested with build 3.3.0.7022 ([downloaded from here](https://ci.appveyor.com/project/gitextensions/gitextensions/branch/master/artifacts)) by running `GitExtension.exe filehistory <path-to-root-of-git-repo>` (since it is a portable build without overriding my current shell menu options), and seems to work properly indeed.
